runtime-decompiler/src/plugins/FernflowerDecompilerWrapper.json

```diff
@@ -2,7 +2,8 @@
   "Name": "Fernflower",
   "WrapperURL": "file:///etc/java-runtime-decompiler/plugins/FernflowerDecompilerWrapper.java",
   "DependencyURL": [
-    "file://${HOME}/.m2/repository/org/jboss/windup/decompiler/decompiler-fernflower/4.2.1.Final/decompiler-fernflower-4.2.1.Final.jar"
+    "file://${HOME}/.m2/repository/org/jboss/windup/decompiler/decompiler-fernflower/4.2.1.Final/decompiler-fernflower-4.2.1.Final.jar",
+    "file://${HOME}/.m2/repository/org/jboss/windup/decompiler/fernflower/windup-fernflower/1.0.0.20171018/windup-fernflower-1.0.0.20171018.jar"
```

Comment:
I'm wondering why the second one,,,

Reply:
Me too, it doesn't work with the first one though.

Reply:
even after merge, I would like to se this resoovled. Please crrreate in issue from it.